cmake/FindcJSON.cmake

```diff
@@ -0,0 +1,22 @@
+# - Try to find cJSON
+# Once done this will define
+#
+#  cJSON_FOUND - cJSON was found
+#  CJSON_INCLUDE_DIRS - cJSON include directories
+#  CJSON_LIBRARIES - libraries needed for linking
+
+find_package(PkgConfig REQUIRED)
```

Comment:
this will abort detection on a system without pkg-config

Reply:
I will change it here as well. I will also replace `CJSON_LIBRARIES` with `cJSON_LIBRARIES` as I have just found out that there is a bug on the master branch. The code currently relies on pkg-config results that use  `CJSON` prefix and ignores results from `find_library`. The same applies for `find_path`.